### PR TITLE
Update Homepage

### PR DESCRIPTION
**Pull request by reetpriye**
As per issue #38 
I've made quite a few changes in the overall UI. The changes include:

- Added favicon
- Added responsive navbar
- Updated homepage(Added overview, motive[with dummy text], join us section)
- Added footer
- Added Routes for all the other pages(i.e. Home, About us, Resources, Broadcasts, Contact us, FAQs and Admin)
I'll improve the UI more in the future after text gets added to the motive section.
If everything seems fine to you, kindly merge my PR.

### CODE REVIEW

File: frontend/src/index.js [hunk 1]
```diff
@@ -1,17 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './app';
-import 'bootstrap/dist/css/bootstrap.min.css';
-import 'font-awesome/css/font-awesome.min.css';
-import 'bootstrap-social/bootstrap-social.css';
-import reportWebVitals from './reportWebVitals';
+import App from './App';
```
Comment (Kajol-Kumari) [inline frontend/src/index.js:3]: it should be `./app` as we are using only lowercase for file name.

File: frontend/src/app.js [hunk 2]
```diff
@@ -1,19 +1,40 @@
...
+import Broadcasts from './components/pages/Broadcasts';
+import Resources from './components/pages/Resources';
+import ContactUs from './components/pages/ContactUs';
+import Admin from './components/pages/Admin';
+import Faqs from './components/pages/Faqs';
+import NotFound from './components/pages/NotFound';
 
-import Header from './components/header_and_footer/Aaryan/header'
-import {BrowserRouter } from 'react-router-dom';
+import './App.css';
```
Comment (Kajol-Kumari) [inline frontend/src/app.js:15]: it should be `./app.css` as we are using only lowercase for the file name.

File: frontend/src/app.js [hunk 3]
```diff
@@ -1,19 +1,40 @@
-import './app.css';
-import Footer from  './components/header_and_footer/Aaryan/footer';
+import { Fragment } from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Footer from './components/Footer';
```
Comment (Kajol-Kumari) [inline frontend/src/app.js:4]: this should be again `./components/footer`

File: frontend/.eslintcache [hunk 4]
```diff
@@ -1 +1 @@
...
...ents\\pages\\AboutUs.js",[],"C:\\Users\\Reet\\Desktop\\Community-Website\\frontend\\src\\App.js",[],"C:\\Users\\Reet\\Desktop\\Community-Website\\frontend\\src\\components\\Footer.js",[],"C:\\Users\\Reet\\Desktop\\Community-Website\\frontend\\src\\components\\Navbar.js",[],"C:\\Users\\Reet\\Desktop\\Community-Website\\frontend\\src\\components\\Button.js",[],"C:\\Users\\Reet\\Desktop\\Community-Website\\frontend\\src\\components\\Dropdown.js",[],"C:\\Users\\Reet\\Desktop\\Community-Website\\frontend\\src\\components\\MenuItems.js",[],["82","83"],"C:\\Users\\Reet\\Desktop\\Community-Website\\frontend\\src\\components\\sections\\home\\Overview.js",[],"C:\\Users\\Reet\\Desktop\\Community-Website\\frontend\\src\\components\\sections\\home\\Motive.js",[],"C:\\Users\\Reet\\Desktop\\Community-Website\\frontend\\src\\components\\pages\\Faqs.js",[],"C:\\Users\\Reet\\Desktop\\Community-Website\\frontend\\src\\components\\sections\\home\\JoinUs.js",[],{"ruleId":"84","replacedBy":"85"},{"ruleId":"86","replacedBy":"87"},{"ruleId":"84","replacedBy":"88"},{"ruleId":"86","replacedBy":"89"},"no-native-reassign",["90"],"no-negated-in-lhs",["91"],["90"],["91"],"no-global-assign","no-unsafe-negation"]
```
Comment (Kajol-Kumari) [inline frontend/.eslintcache:1]: please revert the changes of this file

File: frontend/debug.log [hunk 5]
```diff
@@ -0,0 +1,2 @@
+[1214/075814.418:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (Kajol-Kumari) [inline frontend/debug.log:1]: please remove this file and `package-lock.json` file from this commit as it's not required.

File: frontend/public/favicon/browserconfig.xml [hunk 6]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Kajol-Kumari) [inline frontend/public/favicon/browserconfig.xml:1]: why do we need this file for?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @reetpriye the UI is looking preety good. Just do the requested changes and also 

1. make sure to keep the file names in lower case only and if it's longer, just separate itusing an `_`(underscore)
2. Keep only the required file changes in the commit and remove the rest.

File: frontend/src/components/Navbar.js [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+              FAQs
+            </Link>
+          </li>
+          <li className='nav-item'>
+            <Link
+              to='/admin'
+              className='nav-links-mobile'
+              onClick={closeMobileMenu}
+            >
+              Admin<span>?</span>
```
Comment (Kajol-Kumari) [inline frontend/src/components/Navbar.js:98]: make it `admin` instead of `Admin`

**Comment by reetpriye** [comment]
I've made the necessary modifications. Kindly check.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @reetpriye I can still see redundant files like `frontend/.eslintcache`, `package-lock.json`, `frontend/public/favicon/browserconfig.xml` etc.

Also, you need to make the component from `Navbar` to `navbar` and similarly lowercase the names of the components along with their imports correction. Please let me know if you need any further clarification.
